### PR TITLE
Update Super Tux Kart source to 1.4

### LINKED ISSUE #21: [Enhancement]: UPDATE THE SNAP PACKAGE
### What changes would you like?

An update now that the version 1.4 has been out for some time.

### Any extra information?

This update has Vulkan support, introduces lap trial, new kart, and plenty of other features/improvements, so its important.

### PR DESCRIPTION
**Pull request by goldstar611**
Fixes #21 

- Upstream Source:
   - https://github.com/supertuxkart/stk-code/releases/tag/1.4

If my assumption is correct we may get a test snap built from this PR, based on https://github.com/diddlesnaps/supertuxkart/blob/master/.github/workflows/build-prs.yml

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -142,8 +142,8 @@ parts:
     override-pull: |
       snapcraftctl pull
       sed -i 's|^Icon=.*|Icon=/usr/share/icons/hicolor/128x128/apps/supertuxkart.png|' data/supertuxkart.desktop
-      curl -L -o cacert.patch https://github.com/supertuxkart/stk-code/commit/9bf537ab351f165555048fbc5df88193d8cd70f2.patch
-      patch -Np1 -i cacert.patch
+
+
     parse-info: [usr/share/metainfo/supertuxkart.appdata.xml]
```
Comment (lucyllewy) [inline snap/snapcraft.yaml]: can we drop the two added blank lines?

```suggestion
    parse-info: [usr/share/metainfo/supertuxkart.appdata.xml]
```

**Review by lucyllewy — CHANGES_REQUESTED** [review]
One change I'd like to request, otherwise it looks good to me :-)

File: snap/snapcraft.yaml (hunk 1 above)
Comment (goldstar611) [reply snap/snapcraft.yaml]: This was an artifact of editing on mobile :) It's fixed now.

Comment (lucyllewy) [reply snap/snapcraft.yaml]: Thank you <3

**Review by lucyllewy — APPROVED** [review]

**Pull request merged**